### PR TITLE
Publish 1.10.1

### PR DESCRIPTION
**Pull request by markgrex**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by markgrex** [comment]
@conda-forge-admin please rerender

**Comment by markgrex** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,26 +1,28 @@
...
 
 build:
-  noarch: python
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
-    - python >=3.8
+    - python
```
Comment (markgrex) [inline recipe/meta.yaml:17]: These requirements match the 1.10.1 requirements https://github.com/conda-forge/conjure-python-client-feedstock/blob/86a6620f3e80a051d75131d6e258891381d27da5/recipe/meta.yaml

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,26 +1,28 @@
...
+{% set version = "1.10.1" %}
 
 package:
   name: conjure-python-client
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/c/conjure-python-client/conjure-python-client-{{ version }}.tar.gz
-  sha256: f8ea3fb12851173d45c322299c6f761563b3cdb395ac41ae0a0b1cfc6661c584
+  sha256: d68f19a391a949756b68f46c0b4978efa96f782dca701b2059b7a22967b5cb17
```
Comment (markgrex) [inline recipe/meta.yaml:9]: Taken from https://pypi.org/project/conjure-python-client/1.10.1/#copy-hash-modal-7cbf2d58-34b7-4ed4-a28f-42ab47ec0be5

**Review by ahggns — APPROVED** [review]

**Pull request merged**